batch10/assignment3/syed-bilal.md

```diff
@@ -0,0 +1,11 @@
+2- Using dockerfiles, you donot have to download the image everytime. Once downloaded it ca nbe used multiple times.
```

Comment:
not fully correct, leverage cache means to place the commands which are less likely to be changed on top of Dockerfile